plugins/PaintTool/PaintTool.qml

```diff
@@ -327,7 +327,7 @@ Item
         UM.Label
         {
             anchors.fill: parent
-            text: catalog.i18nc("@label", "Select a single model to start painting")
+            text: catalog.i18nc("@label", "Select a single ungrouped model to start painting")
```

Comment:
Don't we have string freeze? 🥶

Reply:
Yes we do... Maybe we should apply this change only on main ?

Reply:
https://github.com/Ultimaker/Cura/pull/21025